app/frontend/src/features/communities/discussions/Comment.tsx

```diff
@@ -0,0 +1,175 @@
+    gridArea: "avatar",
+    width: "3rem",
+  },
+  nestedCommentsContainer: {
+    "& > * + *": {
+      marginBlockStart: theme.spacing(2),
+    },
+    display: "flex",
+    flexDirection: "column",
+    marginBlockStart: theme.spacing(2),
+    marginInlineStart: theme.spacing(3),
+    "&": {
+      marginInlineStart: `clamp(${theme.spacing(2)}, 5vw, ${theme.spacing(5)})`,
+    },
+  },
+  loadMoreRepliesButton: {
+    alignSelf: "center",
+  },
+}));
+
+export const COMMENT_TEST_ID = "comment";
+
+interface CommentProps {
+  comment: Reply.AsObject;
+  topLevel?: boolean;
+}
+
+export default function Comment({ topLevel = false, comment }: CommentProps) {
+  const classes = useStyles();
+  const { data: user, isLoading: isUserLoading } = useUser(
+    comment.authorUserId
+  );
+
+  const {
+    data: comments,
+    fetchNextPage,
+    hasNextPage,
+    isLoading: isCommentsLoading,
+    isFetching: isCommentsFetching,
+    isFetchingNextPage,
+  } = useThread(comment.threadId, { enabled: topLevel });
+  const isCommentsRefetching = !isCommentsLoading && isCommentsFetching;
+  const showLoadMoreButton = topLevel && hasNextPage;
+
+  const [showCommentForm, setShowCommentForm] = useState(false);
+  const commentFormRef = useRef<HTMLFormElement>(null);
+
+  useEffect(() => {
+    if (showCommentForm && commentFormRef.current) {
+      commentFormRef.current.scrollIntoView({
+        behavior: "smooth",
+        block: "center",
+      });
+    }
+  }, [showCommentForm]);
+
+  const replyDate = timestamp2Date(comment.createdTime!);
+  const postedTime = timeAgo(replyDate, false);
+
+  return (
+    <>
+      <Card className={classes.commentContainer} data-testid={COMMENT_TEST_ID}>
+        <Avatar user={user} className={classes.avatar} />
+        <div className={classes.commentContent}>
+          {isUserLoading ? (
+            <Skeleton />
+          ) : (
+            <Typography variant="body2">
+              {getByCreator(user?.name ?? UNKNOWN_USER)}
+              {` • ${postedTime}`}
+            </Typography>
+          )}
+          {isUserLoading ? <Skeleton /> : <Markdown source={comment.content} />}
+        </div>
+        {topLevel && (
+          <Button
+            className={classes.replyButton}
+            onClick={() => {
+              setShowCommentForm(true);
+            }}
+          >
+            {REPLY}
+          </Button>
+        )}
+      </Card>
+      {isCommentsLoading ? (
+        <CircularProgress />
+      ) : (
+        <div className={classes.nestedCommentsContainer}>
+          {hasAtLeastOnePage(comments, "repliesList") && (
+            <>
+              {comments.pages
+                .flatMap((page) => page.repliesList)
+                .reverse()
+                .map((reply) => {
+                  return <Comment key={reply.threadId} comment={reply} />;
+                })}
+              {!showLoadMoreButton && isCommentsRefetching && (
+                <CircularProgress />
+              )}
```

Comment:
Aha! I wasn't going crazy - this doesn't appear for the first nested comment, because it is inside `hasAtLeastOnePage`